@here/harp-mapview/lib/text/TextElement.ts

```diff
@@ -287,7 +287,7 @@ export class TextElement {
      * A `TextElement` with a higher `renderOrder` will be rendered after a `TextElement` with a
      * lower `renderOrder`.
      */
-    renderOrder?: number = 0;
+    renderOrder = BigInt(0);
```

Comment:
this removes the explicit type information, consider:

renderOrder: bigint = 0n;

instead